From 56c23bc85e0502499fe3263de52c0bef8e60f7aa Mon Sep 17 00:00:00 2001
From: zrj <rimvydas.jasinskas@gmail.com>
Date: Thu, 22 Aug 2019 13:56:08 +0300
Subject: [PATCH 33/52] <sys/fbio.h>: Avoid using vm_offset_t.

 This header is used in userland applications.  Use standard type.
 XXX: struct video_adapter_info has u_int types, limited to low mem?

 No functional change.  The u_long is wide enough to take u_int.
---
 sys/sys/fbio.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/sys/sys/fbio.h b/sys/sys/fbio.h
index e1264c3771..0b01f00cec 100644
--- a/sys/sys/fbio.h
+++ b/sys/sys/fbio.h
@@ -268,11 +268,11 @@ struct video_adapter {
     int			va_io_size;
     int			va_mem_base;
     int			va_mem_size;
-    vm_offset_t		va_window;	/* virtual address */
+    u_long		va_window;	/* virtual address */
     size_t		va_window_size;
     size_t		va_window_gran;
     u_int		va_window_orig;
-    vm_offset_t		va_buffer;	/* virtual address */
+    u_long		va_buffer;	/* virtual address */
     size_t		va_buffer_size;
     int			va_initial_mode;
     int			va_initial_bios_mode;
-- 
2.22.0

